### PR TITLE
Added Coingecko API

### PR DESCRIPTION
**Pull request by arbob**
Free and open API for crypto

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,6 +136,7 @@ APIs
 - [ShapeShift.io](https://shapeshift.io/) - Exchange between cryptocurrencies without an account.  Well documented API for easy use.
 - [Technical Analysis](https://technical-analysis-api.com) - Cryptocurrency prices, technical analysis and sentiment detection.
 - [Poloniex](https://poloniex.com/support/api/) - US based digital asset exchange
+- [Coingecko](https://coingecko.com/en/api) - Free and Public Cryptocurrency API which you can use without an API key.
```
Comment (n0shake) [inline README.md:139]: Thanks for the addition! This should be already alphabetically.

**Review by n0shake — CHANGES_REQUESTED** [review]
Please address the one comment I left.

**Comment by arbob** [comment]
Done.

File: README.md (hunk 1 above)
Comment (arbob) [reply README.md]: done

**Review by n0shake — APPROVED** [review]

**Pull request merged**